src/components/Tooltip.tsx

```diff
@@ -35,6 +35,8 @@ export function Tooltip(props: TooltipProps) {
         {...triggerProps}
         {...(!state.isOpen && typeof title === "string" ? { title } : {})}
         {...tid}
+        // Add display contents to prevent the tooltip wrapping element from short-circuiting inherited styles (i.e. flex item positioning)
+        css={Css.display("contents").$}
```

Comment:
cool! Wasn't aware of that property so checked caniuse on its support: https://caniuse.com/css-display-contents

Maybe worth giving Safari and and Firebox a quick test given the notes?

Reply:
Great suggestion. Will do!

Reply:
I've checked the other browsers, all works fine. For iPads, we need a new pattern for touch devices and tooltips. Right now we show tooltips on hover, but there is no "hover" in touch devices. We'd need an explicit trigger element (like an icon) that when you click it, then it displays the tooltip. Will bring this up with design, but I'm pretty sure they are aware of this issue already - might just not be top of mind.